docs/requirements.txt

```diff
@@ -3,3 +3,4 @@ sphinx_rtd_theme
 numpydoc
 memory_profiler
 pillow
+ipython
```

Comment:
I would have rather killed the `from IPython.display import display` from the examples :)